.pre-commit-config.yaml

```diff
@@ -1,12 +1,43 @@
 repos:
-- repo: git://github.com/antonbabenko/pre-commit-terraform
-  rev: v1.30.0
-  hooks:
-    - id: terraform_fmt
-    - id: terraform_docs
-- repo: https://github.com/pre-commit/pre-commit-hooks
-  rev: v3.0.0
-  hooks:
-    - id: end-of-file-fixer
-    - id: trailing-whitespace
-    - id: no-commit-to-branch
+  - repo: https://github.com/antonbabenko/pre-commit-terraform
+    rev: v1.31.0
+    hooks:
+      - id: terraform_docs
+        args:
+          - --args=--sort-by-required
+      - id: terraform_fmt
+      - id: terraform_tflint
+        alias: terraform_tflint_deep
+        name: terraform_tflint_deep
+        args:
+          - --args=--deep
+      - id: terraform_tflint
+        alias: terraform_tflint_nocreds
+        name: terraform_tflint_nocreds
+      - id: terraform_tfsec
+      - id: terraform_validate
+        exclude: examples
+  - repo: https://github.com/pre-commit/pre-commit-hooks
```

Comment:
There's several python checks available in this repo. You should enable as necessary.